tekore/_convert.py

```diff
@@ -141,6 +141,9 @@ def from_url(url: str) -> Tuple[str, str]:
     """
     Parse type and ID from an URL.
 
+    If the URL includes the tracking token (?si=x)
```

Comment:
Same here, original wording should work. The only thing I'm not keen on was the "GET" part. There is no request involved, so I think just "URL parameters" should be fine.

Reply:
I opted for "Any parameters in the URL will be ignored.", just so it's clear that they won't be parsed at all.

Reply:
That sounds good!